components/typography/styles.js

```diff
@@ -41,26 +41,31 @@ export const bodyStandardStyles = css`
 	}
 `;
 
-export const bodyCompactStyles = css`
-	.d2l-body-compact {
-		font-size: 0.8rem;
-		font-weight: 400;
-		line-height: 1.2rem;
-	}
-	:host([skeleton]) .d2l-body-compact.d2l-skeletize::before {
-		bottom: 0.3rem;
-		top: 0.3rem;
-	}
-	:host([skeleton]) .d2l-body-compact.d2l-skeletize-paragraph-2 {
-		max-height: 2.4rem;
-	}
-	:host([skeleton]) .d2l-body-compact.d2l-skeletize-paragraph-3 {
-		max-height: 3.6rem;
-	}
-	:host([skeleton]) .d2l-body-compact.d2l-skeletize-paragraph-5 {
-		max-height: 6rem;
-	}
-`;
+export const bodyCompactStylize = (selector) => {
```

Comment:
Nit: I'm not too particular about it, but maybe `generateBodyCompactStyles` is more accurate.  🤷‍♂️   That's the naming convention I used recent in the Prism helper when I did this sort of thing.

Reply:
I do like that naming as well.

I'm working on a draft PR specifically for these style changes, since it's a pretty big shift away from what `styles.js` currently has. I don't want conversations around those changes to get buried as in this PR.